internal/cmd/kubernetes.go

```diff
@@ -23,25 +23,35 @@ func clientConfig() clientcmd.ClientConfig {
 	return clientcmd.NewNonInteractiveDeferredLoadingClientConfig(loadingRules, &clientcmd.ConfigOverrides{})
 }
 
-func kubernetesSetContext(name string) error {
+func kubernetesSetContext(cluster, namespace string) error {
```

Comment:
Nice! I better understand how this works now!

A couple tests for this function to show it builds the expected config might be nice, but not blocking.